benchmarks/tables/bpdn-constr-table.jl

```diff
@@ -7,51 +7,101 @@ model, nls_model, sol = bpdn_model(compound, bounds = true)
 
 f = LSR1Model(model)
 λ = norm(grad(model, zeros(model.meta.nvar)), Inf) / 10
-h = NormL1(λ)
+h = NormL0(λ)
 
 verbose = 0 # 10
-ϵ = 1.0e-6
+ν = 1.0
+ϵ = 1.0e-5
+ϵi = 1.0e-5
+ϵri = 1.0e-6
 maxIter = 500
-maxIter_inner = 20
+maxIter_inner = 100
 options =
-  ROSolverOptions(ν = 1.0, ϵa = ϵ, ϵr = ϵ, verbose = verbose, maxIter = maxIter, spectral = true)
-options2 = ROSolverOptions(spectral = false, psb = true, ϵa = ϵ, ϵr = ϵ, maxIter = maxIter_inner)
-options3 = ROSolverOptions(spectral = false, psb = false, ϵa = ϵ, ϵr = ϵ, maxIter = maxIter_inner)
-options4 = ROSolverOptions(spectral = true, ϵa = ϵ, ϵr = ϵ, maxIter = maxIter_inner)
-options6 = ROSolverOptions(
-  ν = 1.0,
+  ROSolverOptions(ν = ν, ϵa = ϵ, ϵr = ϵ, verbose = verbose, maxIter = maxIter, spectral = true)
+optionsbis =
+  ROSolverOptions(ν = ν, ϵa = ϵ, ϵr = ϵ, verbose = verbose, maxIter = maxIter, spectral = true, reduce_TR = false)
+options2 = ROSolverOptions(spectral = false, psb = true, ϵa = ϵi, ϵr = ϵri, maxIter = maxIter_inner)
+options2bis = ROSolverOptions(spectral = false, psb = true, ϵa = ϵi, ϵr = ϵri, maxIter = maxIter_inner, reduce_TR = false)
+options3 = ROSolverOptions(spectral = false, psb = false, ϵa = ϵi, ϵr = ϵri, maxIter = maxIter_inner)
+options3bis = ROSolverOptions(spectral = false, psb = false, ϵa = ϵi, ϵr = ϵri, maxIter = maxIter_inner, reduce_TR = false)
+options4 = ROSolverOptions(spectral = true, ϵa = ϵi, ϵr = ϵri, maxIter = maxIter_inner)
+options4bis =
```

Comment:
Let's give all these options descriptive names and avoid "bis" (it's French and will confuse non-French speakers).